apps/projects/app/components/Content/IssueDetail/DetailsCard.js

```diff
@@ -42,69 +45,49 @@ const DetailsCard = ({ issue }) => {
     grid: PropTypes.string.isRequired,
   }
 
-  const OpenedBy = ({ author }) => (
-    <Text size="large" css="display: flex; align-items: center">
-      <img src={author.avatarUrl} alt="user avatar" css="margin: 8px; width: 24px; border-radius: 50%;" />
-      <Link
-        href={author.url}
-        target="_blank"
-        style={{ textDecoration: 'none', color: `${theme.link}` }}
-      >
-        {author.login}
-      </Link>
-    </Text>
-  )
-  OpenedBy.propTypes = userGitHubShape
-
   const SummaryTable = ({ issue }) => (
     <div css={`
       display: grid;
-      grid-template-columns: repeat(${issue.hasBounty ? '3' : '4'}, 1fr);
+      grid-template-columns: repeat(${issue.hasBounty ? '2' : '3'}, 1fr);
       grid-template-rows: auto;
       ${issue.hasBounty ? `
         grid-template-areas:
-          'deadline exp openedby' 'description description description'
+          'deadline exp' 'description description'
         ` : `
         grid-template-areas:
-          'description description description openedby'
+          'description description description'
```

Comment:
Seems the same as having one column, or just getting rid of the grid when the issue has a bounty.

Reply:
True! Fixed.